lib/src/models/multireddit.dart

```diff
@@ -299,13 +263,13 @@ class Multireddit extends RedditBase with RedditBaseInitializedMixin {
   ///
   /// [subreddit] is the name of the [Subreddit] to be added to this [Multireddit].
   Future add(/* String, Subreddit */ subreddit) async {
-    subreddit = _subredditNameHelper(subreddit);
+    final subredditName = _subredditNameHelper(subreddit);
     if (subreddit == null) return;
     final url = apiPath[_kMultiredditUpdate]
         .replaceAll(_userRegExp, _author)
         .replaceAll(_multiredditRegExp, displayName)
         .replaceAll(_subredditRegExp, subreddit);
-    await reddit.put(url, body: {'model': '{"name" : "$subreddit"}'});
+    await reddit.put(url, body: {'model': '{"name" : "$subredditName"}'});
     await refresh();
```

Comment:
We probably shouldn't do a refresh here since it's quite a heavy operation. I think what would be best here is to insert a `SubredditRef` into our list of subreddits that are part of this multireddit instead of re-requesting all the data for the object.